test/org/zaproxy/zap/extension/ascanrulesBeta/SourceCodeDisclosureCVE20121823UnitTest.java

```diff
@@ -73,6 +74,19 @@ public void shouldNotTargetNonPhpTechs() throws Exception {
     }
 
     @Test
+    public void shouldIgnoreNonTextResponses() throws Exception {
+        // Given
+        HttpMessage message = getHttpMessage("/");
+        message.setResponseHeader("HTTP/1.1 200 Ok \\r\\nConnection: close\\r\\n\\r\\n");
```

Comment:
Is the re-set of the header really needed? The base message already is a 200 OK. (The `\` are being escaped, not on purpose?)

Reply:
Huh, not I hadn't escaped those on purpose. Seems to be something related to copy/paste in Eclipse Oxygen. Will watch for that in the future.

Update inbound.

Reply:
Yeah, that's a new Oxygen feature/option :)